### ISSUE TITLE
Files missing in the -docs tarball

### ISSUE THREAD

**Issue by GoogleCodeExporter** [keithgg/puddletag#259]
```
Hi, in the docs tarball there are some file missing:

$ grep -A6 ^html\: Makefile 
html:
        python2 createnews.py
        $(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
        echo
        python2 createnews.py fixnews
        python2 fix_html.py "_build/html"
        echo "Build finished. The HTML pages are in $(BUILDDIR)/html."

but createnews.py and fix_html.py are missing from the tarball, and hence the 
html target cant be executed.
```

Original issue reported on code.google.com by `sandro.tosi` on 7 Apr 2015 at 8:55

**Comment by sandrotosi** [comment]
now that we have all in git, it would be nice if code and doc could be merged and so that we could get a full release all the times :)

**Comment by sandrotosi** [comment]
these are the files I have to add to be able to compile the documentation as released in the 1.0.5 tarball:

 _templates/my_sidebar.html |    5 +++++
 checksums.txt              |   23 +++++++++++++++++++++++
 index.txt                  |   45 +++++++++++++++++++++++++++++++++++++++++++++

**Comment by keithgg** [comment]
Files should be all there. Re-open if not.
